docs/classad-attributes/machine-classad-attributes.rst

```diff
@@ -328,8 +328,9 @@ Machine ClassAd Attributes
     *condor_starter* has exited.
 
 :classad-attribute-def:`JobBusyTimeCount`
-    attribute. This is also the total number times a
-    *condor_starter* has exited.
+    The total number of jobs used to calculate the :ad-attr:`JobBusyTimeAvg`
+    attribute. This is also the the total number times a *condor_starter*
+    has exited.
```

Comment:
This is an amount of *time*, not a number of jobs.

Reply:
This was the original definition with minimal changes prior to being stomped, and I think this is a number of jobs since it is the number of jobs used to calculate that JobBusyTime. I think this needs some digging for certainty. TJ may know